src/components/ModuleFields/Configurator/File.vue

```diff
@@ -45,6 +45,138 @@
         {{ $t('kind.file.view.showName') }}
       </b-form-checkbox>
     </b-form-group>
+
+    <hr>
```

Comment:
We should only show the style section if the view mode supports the styling (guessing the list doesn't), also lets rename `Style` to `Preview style`